resources/views/pages/partials/mute-promotions/logs.blade.php

```diff
@@ -15,3 +15,5 @@
         </tr>
     @endforeach
 </table>
+
+{{$rows->links()}}
```

Comment:
This throws an error `Method Illuminate\Database\Eloquent\Collection::links does not exist.` when being rendered from the user view.